settings.php

```diff
@@ -62,6 +62,12 @@
     $description = new lang_string('settings:guidancepage_help', 'tool_mfa');
     $settings->add(new admin_setting_confightmleditor('tool_mfa/guidancecontent', $name, $description, '', PARAM_RAW));
 
+    $name = new lang_string('settings:guidancefiles', 'tool_mfa');
```

Comment:
This is a pita and has come up a few times. I have seen how to workaround this "properly" by sub classing admin_setting_confightmleditor but can't remember where I saw it. I've made a tracker https://tracker.moodle.org/browse/MDL-72975